### ISSUE TITLE
Linphone crashes when ending a regular phone call

### ISSUE THREAD

**Issue by marmistrz** [BelledonneCommunications/linphone-android#54]
```
04-18 12:44:48.392  9077  9077 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getString(int)' on a null object reference
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at org.linphone.ContactsManager.getString(ContactsManager.java:487)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at org.linphone.ContactsManager.fetchContactsSync(ContactsManager.java:427)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at org.linphone.ContactsManager.onChange(ContactsManager.java:109)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.database.ContentObserver.onChange(ContentObserver.java:145)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.database.ContentObserver$NotificationRunnable.run(ContentObserver.java:216)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5461)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
04-18 12:44:48.392  9077  9077 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:102)
```

**Comment by DatBewar** [comment]
Hello, on which version of Linphone did you had this crash?
It's seems to be resolved with this commit: 161ac91de19cfe6bde86713f921a4a0821f0f9df

**Comment by marmistrz** [comment]
Linphone 3.2.5

**Comment by DatBewar** [comment]
Ok, this is fixed for 3.2.6, you can find the beta on google-store, the apk on our website https://www.linphone.org/releases/android/linphone-latest-beta.apk
I'm currently at the search of time to deploy our F-Droid repo.
